docs/styling.md

```diff
@@ -0,0 +1,69 @@
+# Styling
+
+This app implements the [USDR Design System](https://www.figma.com/file/HWu4iIjcoX2txuN543qzIB/USDR-Design-System?type=design&node-id=101%3A167&mode=design&t=8mml86X91f9FzVAE-1), defined in Figma.
+
+## Overview
+
+This app is built with [Bootstrap Vue](https://bootstrap-vue.org/).
+
+The implementation of the USDR design system primarily overrides Bootstrap's theme colors and fonts. Bootstrap provides a small set of [theme color definitions](https://getbootstrap.com/docs/5.0/customize/css-variables/#root-variables). This is the primary way the USDR brand is expressed in this app through Bootstrap.
```

Comment:
You asked for nit-picks, so here's one 😄 – I think it's nice when markdown/doc files try to conform line-length to the same number of characters as the rest of the code base (90/100 in our case). It doesn't make a difference when viewing the rendered docs, but greatly improves readability when viewing in a text editor/IDE where long lines don't automatically wrap.

E.g. something like
```suggestion
The implementation of the USDR design system primarily overrides Bootstrap's theme colors and fonts. 
Bootstrap provides a small set of [theme color definitions](https://getbootstrap.com/docs/5.0/customize/css-variables/#root-variables). 
This is the primary way the USDR brand is expressed in this app through Bootstrap.
```

Reply:
Thank you! My editor auto-wraps lines so I missed this. Some of the lines with links get long but otherwise added returns after each long-ish sentence, please take a look~